sunpy/map/mapsequence.py

```diff
@@ -271,6 +271,12 @@ def plot(self, axes=None, resample=None, annotate=True,
         plot_function : function
             A function to be called as each map is plotted.
             For more information see `sunpy.visualization.animator.MapSequenceAnimator`.
+        norm : 
+            Normalization used in scaling the plot.
+            This will override the predefined value in ``plot_settings['norm']``.
+        cmap : 
```

Comment:
The type is missing, that needs to be filled in.

Reply:
I don't know what they are. You will need to work it out by either checking the matplotlib docs or running the code and checking the types manually.